### PR TITLE
Fix issue with k8s.io/docs/concepts/policy/limit-range/

### PR DESCRIPTION
**Pull request by pjhwa**
from #20016

### CODE REVIEW

**Comment by pjhwa** [comment]
Preview: https://deploy-preview-20017--kubernetes-io-master-staging.netlify.com/docs/concepts/policy/limit-range/

File: content/en/docs/concepts/policy/limit-range.md [hunk 1]
```diff
@@ -207,11 +207,11 @@ kubectl get po/busybox1 -o json | jq ".spec.containers[3].resources"
 ```
 
 -  The `busybox-cnt04` Container inside `busybox1` define neither `limits` nor  `requests`.
--  The Container do not define a limit section, the default limit defined in the limit-mem-cpu-per-container LimitRange is used to fill its request
-   `limits.cpu=700m and` `limits.memory=900Mi` .
--  The Container do not define a request section, the defaultRequest defined in the `limit-mem-cpu-per-container` LimitRange is used to fill its request section requests.cpu=110m and requests.memory=111Mi
+-  The Container do not define a limit section, the default limit defined in the `limit-mem-cpu-per-container` LimitRange is used to fill its request
```
Comment (sftim) [inline content/en/docs/concepts/policy/limit-range.md:210]: If you also want to fix a related error:
```suggestion
-  The Container does not define a limit section, the default limit defined in the `limit-mem-cpu-per-container` LimitRange is used to fill its request
```

**Comment by sftim** [comment]
@pjhwa there is a merge conflict; could you take a look?

**Comment by pjhwa** [comment]
> @pjhwa there is a merge conflict; could you take a look?

Actually my fixes applied for the "Limiting Container compute resources" section. So, if the section has been removed, my fixes are no longer needed I think. So, can I just close my PR?

**Comment by sftim** [comment]
I'm going to say yes and 
/close

If you did want to reopen this then of course feel free.

**Pull request closed without merging**